config/routes.rb

```diff
@@ -27,4 +27,13 @@
   get "/apple-touch-icon-76x76.png", to: "icon_redirects#apple_old_size_icon"
   get "/apple-touch-icon-60x60.png", to: "icon_redirects#apple_old_size_icon"
   get "/apple-touch-icon-precomposed.png", to: "icon_redirects#apple_old_size_icon"
```

Comment:
I notice some of these return 404s:

```
➜  ~ curl -sI https://www.gov.uk/apple-touch-precomposed.png | grep HTTP
HTTP/2 404
➜  ~ curl -sI https://www.gov.uk/apple-touch-icon-76x76.png | grep HTTP
HTTP/2 404
➜  ~ curl -sI https://www.gov.uk/favicon.ico | grep HTTP
HTTP/2 301
```

Which makes me wonder if there's a part of puppet that needs to be kept in sync with this. I'll just have a look and comment further with my findings. I wouldn't want you to add these and find out that no-one can access them.

Reply:
🤔

Reply:
In a rather grim TIL moment I've discovered these assets are put into the Publishing API 🤯  via: https://github.com/alphagov/static/blob/fc25d7873c7aec702805abd7c5e20792e8989d5e/lib/tasks/publishing_api.rake#L48-L83 meaning we have content items like: https://www.gov.uk/api/content/favicon.ico

It looks like you'll need to add the new items to this list too and run the rake tasks when they're published. There's surely a nicer way these could have been done 🤔 

Btw the first Item I checked https://www.gov.uk/apple-touch-precomposed.png I had the URL wrong it's fine for https://www.gov.uk/apple-touch-icon-precomposed.png

Reply:
I spotted this a few days ago, but couldn't make sense of the _why_ – I'll add them there too.

Reply:
Thanks for the support on this – I added https://github.com/alphagov/static/pull/2375/commits/d20c1c23d019a1fe53f2fae4b9060b11da26458c to prepare the new assets to be published via Publishing API